pandas-stubs/core/indexes/accessors.pyi

```diff
@@ -157,7 +157,7 @@ _DTRoundingMethodReturnType = TypeVar(
     TimedeltaSeries,
     TimestampSeries,
     DatetimeIndex,
-    # TimedeltaIndex
```

Comment:
looks like a leftover

Reply:
It was a leftover from the "right" way to do it - that's what is put back in this PR!

Reply:
oh, stupid me :)